bindings/sampling-heap-profiler.cc

```diff
@@ -81,13 +81,26 @@ NAN_METHOD(GetAllocationProfile) {
   info.GetReturnValue().Set(TranslateAllocationProfile(root));
 }
 
+NAN_METHOD(GetAllocationProfileProto) {
+  if (info.Length() != 1 || !info[0]->IsNumber()) {
+    return Nan::ThrowTypeError("First argument type must be Integer.");
+  }
+  int64_t startTimeNanos = info[0].As<Integer>()->IntegerValue();
```

Comment:
Does the function take start time to calculate the duration? I'd pass in the duration if so.

Reply:
Duration is not set for heap profiles. Should I set the duration? If so, to what?